src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Subscriptions.cs

```diff
@@ -0,0 +1,239 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Maestro.ScenarioTests.ObjectHelpers;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Maestro.ScenarioTests
+{
+    public class ScenarioTests_Subscriptions : MaestroScenarioTestBase
```

Comment:
Can you put the [Category="ScenarioTests"] that I put on the other scenario tests? It will help make sure we only run stuff at the right time if we ever have other assemblies.